internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -292,18 +305,25 @@ func validatingToWebhookDescription(webhook admissionregv1.ValidatingWebhook) op
 		TimeoutSeconds:          webhook.TimeoutSeconds,
 		AdmissionReviewVersions: webhook.AdmissionReviewVersions,
 	}
+	if len(description.AdmissionReviewVersions) == 0 {
+		description.AdmissionReviewVersions = defaultAdmissionReviewVersions
+	}
+
```

Comment:
Can we have a check here to see if `description.SideEffects` is null, if so set it to [`SideEffectClassNone`](https://pkg.go.dev/k8s.io/api/admissionregistration/v1?tab=doc#SideEffectClass)? Because having it null causes installplan to fail.

Reply:
I’ll do this in a follow-up. Ideally I wouldn’t be modifying admission review versions in this PR either since they’re unrelated to the deployment name fix, so I’ll break that out too.